### ISSUE TITLE
The xAxis tooltip on the radar chart is not applicable to showTooltipOn: "Always". What should I do?

### ISSUE THREAD

**Issue by shinkangwon** [amcharts/amcharts5#1777]
This is the code that draws the Rader Chart of the official document.
How do I apply it here?

// Create root element
// https://www.amcharts.com/docs/v5/getting-started/#Root_element
var root = am5.Root.new("chartdiv");

// Set themes
// https://www.amcharts.com/docs/v5/concepts/themes/
root.setThemes([
  am5themes_Animated.new(root)
]);

// Create chart
// https://www.amcharts.com/docs/v5/charts/radar-chart/
var chart = root.container.children.push(am5radar.RadarChart.new(root, {
  panX: false,
  panY: false,
  wheelX: "panX",
  wheelY: "zoomX"
}));

// Add cursor
// https://www.amcharts.com/docs/v5/charts/radar-chart/#Cursor
var cursor = chart.set("cursor", am5radar.RadarCursor.new(root, {
  behavior: "zoomX"
}));

cursor.lineY.set("visible", false);

// Create axes and their renderers
// https://www.amcharts.com/docs/v5/charts/radar-chart/#Adding_axes
var xRenderer = am5radar.AxisRendererCircular.new(root, {});
xRenderer.labels.template.setAll({
  radius: 10
});

var xAxis = chart.xAxes.push(am5xy.CategoryAxis.new(root, {
  maxDeviation: 0,
  categoryField: "country",
  renderer: xRenderer,
  tooltip: am5.Tooltip.new(root, {})
}));

var yAxis = chart.yAxes.push(am5xy.ValueAxis.new(root, {
  renderer: am5radar.AxisRendererRadial.new(root, {})
}));

// Create series
// https://www.amcharts.com/docs/v5/charts/radar-chart/#Adding_series
var series = chart.series.push(am5radar.RadarLineSeries.new(root, {
  name: "Series",
  xAxis: xAxis,
  yAxis: yAxis,
  valueYField: "litres",
  categoryXField: "country",
  tooltip:am5.Tooltip.new(root, {
    labelText:"{valueY}"
  })
}));

series.strokes.template.setAll({
  strokeWidth: 2
});

series.bullets.push(function () {
  return am5.Bullet.new(root, {
    sprite: am5.Circle.new(root, {
      radius: 5,
      fill: series.get("fill")
    })
  });
});

// Set data
// https://www.amcharts.com/docs/v5/charts/radar-chart/#Setting_data
var data = [{
  "country": "Lithuania",
  "litres": 501
}, {
  "country": "Czechia",
  "litres": 301
}, {
  "country": "Ireland",
  "litres": 266
}, {
  "country": "Germany",
  "litres": 165
}, {
  "country": "Australia",
  "litres": 139
}, {
  "country": "Austria",
  "litres": 336
}, {
  "country": "UK",
  "litres": 290
}, {
  "country": "Belgium",
  "litres": 325
}, {
  "country": "The Netherlands",
  "litres": 40
}];
series.data.setAll(data);
xAxis.data.setAll(data);

// Animate chart and series in
// https://www.amcharts.com/docs/v5/concepts/animations/#Initial_animation
series.appear(1000);
chart.appear(1000, 100);

I'd appreciate it if you could let me know.

**Comment by shinkangwon** [comment]
I am not familiar with English, so I used the translator. I apologize if my question seems rather rude.

**Comment by shinkangwon** [comment]
Yes. I want all tooltips for the 'country' value in the CategoryField to always be displayed.

**Comment by martynasma** [comment]
In that case you need to add a separate `Tooltip` object to each bullet:

```JavaScript
series.bullets.push(function(root) {
  return am5.Bullet.new(root, {
    sprite: am5.Circle.new(root, {
      radius: 5,
      fill: series.get("fill"),
      tooltipText: "{categoryX}: {valueY}",
      showTooltipOn: "always",
      tooltip: am5.Tooltip.new(root, {})
    })
  });
});
```

**Comment by shinkangwon** [comment]
I'm sorry. I guess I didn't explain enough.
What I'm trying to do is make sure that only the tooltips marked with the red box in the attached picture are always visible. The value of Y will remain in hover form.
To apply the method you taught me,
```
var xAxis = chart.xAxes.push(am5xy.CategoryAxis.new(root, {
                  maxDeviation: 0,
                  categoryField: "country",
                  renderer: xRenderer,
                  //tooltip: am5.Tooltip.new(root, {})
                  bullet: function (root, axis, dataItem) {
                          return am5xy.AxisBullet.new(root, {
                                    sprite: am5.Circle.new(root, {
                                              radius: 0,
                                              fill: am5.color(0x000000),
                                              tooltipText: "{category}",
                                              showTooltipOn: "always",
                                              tooltip: am5.Tooltip.new(root, {})
                                   })
                          });
                  }
           }));
```
Is this right?
Can you tell me if there's anything wrong?
![2024-10-30 10 31 16](https://github.com/user-attachments/assets/0efc2f70-0473-4d70-a795-992dd476ff8e)

**Comment by martynasma** [comment]
So you want tooltips on categories always be displayed or on click? I'm not sure I understand.

**Comment by shinkangwon** [comment]
For example 
https://codepen.io/pen
In this example, when you click Title, it becomes tooltip.show(). 
Click again to become tooltip.hide().  
Here, I would like to put show TooltipOn: "always" in the tooltip option that appears as tooltip.show() when clicked. Also, as of now, only one tooltip is showing, but I want to make the whole category show.

**Comment by martynasma** [comment]
I propose different approach: using axis bullets:

```JavaScript
xAxis.set("bullet", function (root, axis, dataItem) {
  return am5xy.AxisBullet.new(root, {
    location: 0.5,
    sprite: am5.Circle.new(root, {
      radius: 0,
      fill: am5.color(0xff0000),
      fillOpacity: 0,
      tooltip: am5.Tooltip.new(root, {}),
      tooltipText: "Toltip:\n{category}"
    })
  });
});

var isComp = false;
title.events.on("click", function() {
  if (!isComp) {
    am5.array.each(xAxis.dataItems, function(item) {
      item.get("bullet").get("sprite").showTooltip();
    });
    isComp = true;
  } else {
    am5.array.each(xAxis.dataItems, function(item) {
      item.get("bullet").get("sprite").hideTooltip();
    });
    isComp = false;
  }
});
```

Example:
https://codepen.io/team/amcharts/pen/xxvjPRX/3ffafddfed5f014dbf137cf4a68d6fd3?editors=0010

**Comment by shinkangwon** [comment]
Yeah, this is what I wanted.
Thank you. It worked out thanks to you.
